packages/driver/cypress/integration/e2e/multi-domain/multi_domain_cypress_api.spec.ts

```diff
@@ -0,0 +1,253 @@
+// @ts-ignore / session support is needed for visiting about:blank between tests+
+describe('multi-domain Cypress API', { experimentalSessionSupport: true, experimentalMultiDomain: true }, () => {
+  beforeEach(() => {
+    cy.visit('/fixtures/multi-domain.html')
+    cy.get('a[data-cy="multi-domain-secondary-link"]').click()
+  })
+
+  context('Commands', () => {
+    context('add', () => {
+      it('adds a custom command', () => {
+        cy.switchToDomain('foobar.com', () => {
+          // @ts-ignore
+          Cypress.Commands.add('foo', () => {
+            return cy.wrap('bar')
+          })
+
+          // @ts-ignore
+          cy.foo().then((shouldBeBar) => {
+            expect(shouldBeBar).to.equal('bar')
+          })
```

Comment:
```suggestion
          cy.foo().should('equal', 'bar')
```

Can shorten this and the other ones.

Reply:
addressed in ddf1daf3bf